drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -122,18 +130,6 @@ def active
     operating_end_date >= Date.current
   end
 
-  def enrollments_including_wip
-    enrollment_ids = enrollments.pluck(:id)
-    wip_ids = wip_enrollments.pluck(:id)
-    Hmis::Hud::Enrollment.where(id: enrollment_ids + wip_ids)
-  end
-
-  def households_including_wip
-    household_ids = enrollments_including_wip.pluck(:household_id)
-
-    Hmis::Hud::Household.where(HouseholdID: household_ids, data_source_id: data_source_id)
-  end
```

Comment:
I like that this code is deleted, but its resulting in the same household being resolved repeatedly (once per enrollment in the household). That's why we had the `pluck` and lookup in the first place. Maybe there is a cleaner way to handle it?

<img width="1117" alt="Screen Shot 2023-06-28 at 5 03 49 PM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/77ca9781-8a99-42c1-9a87-f98f99dcf18f">

Reply:
Good catch! I added a `distinct` to that association. That should fix the duplicates

Reply:
Ok great, thanks